### PR TITLE
node_e2e_installer: be use runc/crun from PATH

### PR DESCRIPTION
**Pull request by haircommander**
the get script only installs runc if it can't be found in path, which is
good behavior. Use the one found in path when installing the drop-in
files for node e2e tests

Signed-off-by: Peter Hunt <pehunt@redhat.com>

#### What type of PR is this?

/kind bug
#### Does this PR introduce a user-facing change?

```release-note
none
```

### CODE REVIEW

**Comment by haircommander** [comment]
related to https://github.com/kubernetes/test-infra/pull/26998

File: scripts/node_e2e_installer [hunk 1]
```diff
@@ -30,20 +30,22 @@ install_crio() {
     # Setup log level
     echo "CONTAINER_LOG_LEVEL=debug" >>/etc/sysconfig/crio
 
+    runc_path=$(command -v runc || true)
```
Comment (saschagrunert) [inline scripts/node_e2e_installer:33]: Do we wanna fail if it's not in $PATH? Or: do we wanna fallback to `/usr/local/bin/runc` in that case?

Comment (saschagrunert) [reply scripts/node_e2e_installer:33]: @haircommander yeah CRI-O should find it in $PATH as well :+1:

**Review by saschagrunert — APPROVED** [review]
/lgtm

**Pull request merged**